commands/builds/output.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+let cli     = require('heroku-cli-util');
+let request = require('request');
+
+module.exports = {
+  topic: 'builds',
+  command: 'output',
+  needsAuth: true,
+  needsApp: true,
+  description: 'show build output',
+  help: 'Show build output for a Heroku app',
+  variableArgs: true,
+  run: cli.command(function (context, heroku) {
```

Comment:
Here, it might be a good idea to name the anonymous function and define it at the bottom (allowing it to be hoisted).

```js
run: cli.command(showOutput)
...
function showOutput(context, heroku) {
...
}
```

Separates the layers of abstraction and provides better stack traces.